Userland/Applications/Browser/BookmarksBarWidget.cpp

```diff
@@ -1,12 +1,15 @@
 /*
+ * Copyright (c) 2020, Emanuel Sprung <emanuel.sprung@gmail.com>
+ * Copyright (c) 2022, networkException <networkexception@serenityos.org>
+ * Copyright (c) 2023, Cameron Youell <cameronyouell@gmail.com>
```

Comment:
Disregard the previous comment regarding copyrights, these additions just need to be moved to the previous commit.